specs/data/boxTexturedUnoptimized/CesiumTexturedBoxTest.gltf

```diff
@@ -98,6 +98,26 @@
             "uri": "CesiumTexturedBoxTest.bin"
         }
     },
+    "cameras": {
+        "camera_0": {
+            "perspective": {
+                "aspect_ratio": 1.5,
```

Comment:
This should be `aspectRatio`.

See https://github.com/KhronosGroup/glTF/blob/master/specification/README.md#perspectiveaspectratio

Where did you copy this from?  I will update it.